GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -321,7 +321,8 @@ public static ArgumentString StashSaveCmd(bool untracked, bool keepIndex, string
                 { isPartialStash, "push", "save" },
                 { untracked && GitVersion.Current.StashUntrackedFilesSupported, "-u" },
                 { keepIndex, "--keep-index" },
-                message.QuoteNE(),
+                { isPartialStash && !message.IsNullOrEmpty(), "-m" },
+                { !message.IsNullOrEmpty(), message.Quote() },
```

Comment:
message.QuoteNE(), 
for the last line, QuouteNE will quote not null or empty

Reply:
It makes it quite clear on what is being done and since we know message isn't empty, we use Quote.   I know QuoteNE would have worked.  I suggested this code to make sure it was clear.

Reply:
This is why I used IsNullOrEmpty.  If It is empty then don't even set a message.  Messages are optional when saving a stash.  Recommended, but optional.  If the message ="               ", why include it?

Reply:
` ` is neither `null` nor empty... :)
Is `string.IsNullOrWhitespace` more appropriate?

Reply:
👍 DURP!  Thanks.  

@thewholuver94   Please change to IsNullOrWhitespace